site/cds_rdm/tasks.py

```diff
@@ -140,3 +147,224 @@ def sync_users():
     except Exception as e:
         db.session.rollback()
         current_app.logger.exception(e)
+
+
+@shared_task()
+def sync_local_accounts_to_names(since, user_id=None):
+    """Syncs local accounts to names vocabulary.
+
+    If the name is marked as "unlisted" - meaning it is deprecated - we still update it with the new values, if needed.
+    """
+    prop_values = ["group", "department", "maibox", "section"]
+    service = current_service_registry.get("names")
+
+    def _check_if_update_needed(user, name, is_orcid=False):
+        """Check if the name needs to be updated."""
+
+        updated = False
+        updated_name = {**name}
+
+        if not is_orcid:
```

Comment:
I guess this means orcid takes precedence as a source of truth when it comes to updates
I am not sure though if thats what we want to achieve, I think CERN will be more reliable than orcid to be up to date regarding name and surname, no?